python/cudf_polars/cudf_polars/containers/column.py

```diff
@@ -33,10 +34,32 @@ def __init__(
         null_order: plc.types.NullOrder = plc.types.NullOrder.BEFORE,
     ):
         self.obj = column
+        self.is_scalar = self.obj.size() == 1
+        if self.is_scalar:
+            is_sorted = plc.types.Sorted.YES
         self.is_sorted = is_sorted
         self.order = order
         self.null_order = null_order
 
+    @property
+    def obj_scalar(self) -> plc.Scalar:
+        """
+        View the column object as a pylibcudf Scalar.
+
+        Returns
+        -------
+        pylibcudf Scalar object.
+
+        Raises
+        ------
+        RuntimeError if the column is not length-1.
+        """
+        if not self.is_scalar:
+            raise RuntimeError(
+                f"Cannot convert a column of length {self.obj.size()} to scalar"
+            )
+        return plc.copying.get_element(self.obj, 0)
```

Comment:
Is this column object mutable and if not is this a cacheable result? `get_element` involves a kernel launch.

Reply:
Everything is functional, so columns should be treated as immutable, so yes, we could do that.

Reply:
Added a note that the column is immutable, and cached this property.